package.json

```diff
@@ -103,5 +103,9 @@
     "chai-enzyme": {
       "cheerio": "1.0.0-rc.10"
     }
+  },
+  "dependencies": {
+    "@leafygreen-ui/skeleton-loader": "^2.0.11",
```

Comment:
Can we add this dependency to the `packages/compass-components/package.json` instead?
Similar with the `mongodb-mql-engines` dependency, let's have that in the `compass-indexes` `package.json`